src/platform/CMakeLists.txt

```diff
@@ -12,16 +12,23 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-add_library(platform STATIC
-  platform_linux.cpp
-  platform_shared.cpp
-  platform_unix.cpp)
 
-target_link_libraries(platform
-  qemu_backend
-  libvirt_backend
-  lxd_backend
-  journaldlogger)
+if(LINUX)
+  add_library(platform STATIC
+    platform_linux.cpp
+    platform_shared.cpp
+    platform_unix.cpp)
+
+  target_link_libraries(platform
+    libvirt_backend
+    lxd_backend
+    journaldlogger)
+endif()
+
+if(LINUX OR ENABLE_QEMU)
+  target_link_libraries(platform
+    qemu_backend)
+endif()
```

Comment:
Again, this is more for fixing conflicts in the private branch.  I kept it like this since it will *always* be enabled on Linux, but conditional elsewhere.

Reply:
No longer applicable.